src/app/images/components/ImagesForms/ImagesForms.test.tsx

```diff
@@ -1,55 +1,61 @@
-import configureStore from "redux-mock-store";
+import { describe } from "vitest";
```

Comment:
*Technically* you don't need this import since `describe` and other vitest basics are auto-imported, but it's just a nitpick